src/rules/newline-after-import.js

```diff
@@ -113,6 +114,33 @@ after ${type} statement not followed by another ${type}.`,
       }
     }
 
+    function commentAfterImport(node, nextComment) {
+      const options = context.options[0] && 'count' in context.options[0] ? context.options[0] : { count: 1 };
```

Comment:
i'd hope we can do this defaulting in only one place

Reply:
Like a global variable? or passing as function parameters?

Reply:
not global, but a rule-level variable - or passing in, yep :-)